bvm/ballerina-core/src/main/java/org/ballerinalang/util/TableIterator.java

```diff
@@ -325,4 +421,25 @@ private void generateColumnDefinitions() {
             columnDefs.add(def);
         }
     }
+
+    private static class ArrayElementNullabilityAttributes {
```

Comment:
Shall we rename this to "ArrayElementAttributes" as it is not all about null, but also the array element type.